extensions/survey/src/survey-preference.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component } from "@k8slens/extensions";
+import React from "react";
+import { observer } from "mobx-react";
+import { SurveyPreferencesStore } from "./survey-preferences-store";
+
+@observer
+export class SurveyPreferenceInput extends React.Component<{survey: SurveyPreferencesStore}, {}> {
+  render() {
+    const { survey } = this.props;
+
+    return (
+      <Component.Checkbox
+        label="Allow in-app surveys"
+        value={survey.enabled}
+        onChange={v => { survey.enabled = v; }}
```

Comment:
Other places don't use all of those, I don't think it is needed here either.
```suggestion
        onChange={v => survey.enabled = v}
```

Reply:
Changed